app/javascript/mastodon/features/account_timeline/index.js

```diff
@@ -16,7 +16,7 @@ import { fetchAccountIdentityProofs } from '../../actions/identity_proofs';
 import MissingIndicator from 'mastodon/components/missing_indicator';
 import TimelineHint from 'mastodon/components/timeline_hint';
 import { me } from 'mastodon/initial_state';
-import { connectUserTimelineStream } from '../../actions/streaming';
+import { connectTimeline, disconnectTimeline } from 'mastodon/actions/timelines';
```

Comment:
`connectTimeline` doesn't actually connect to a timeline, does it? From what I can tell it's only used to signal whether the timeline is connected or not, but doesn't set up streaming or polling.

Reply:
That is the point. All that we need is that `insertIfOnline` from the composer, and for that we need to mark the timeline as online. Making an actual second connection to the user stream is entirely pointless, the app is already connected to the user stream as long as it's open.

Reply:
hmm, alright